apps/web/src/pages/settings/tabs/ApiKeysCard.cy.tsx

```diff
@@ -0,0 +1,132 @@
+import { ApiKeysCard } from '.';
+import { TestWrapper } from '../../../testing';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { IEnvironment } from '@novu/shared';
+
+const myEnvironments: IEnvironment = {
+  _id: '633a68831c6bf8ce582c923e',
+  name: 'test',
+  _organizationId: '633a68831c6bf8ce582c9238',
+  identifier: '38rTyAK2BJ7Z',
+
+  widget: {
+    notificationCenterEncryption: false,
+  },
+};
+
+const currentEnvironment = {
+  createdAt: '2022-10-03T04:43:47.460Z',
+  id: '633a68831c6bf8ce582c923e',
+  identifier: '38rTyAK2BJ7Z',
+  name: 'Development',
+  updatedAt: '2022-10-03T04:43:47.460Z',
+  widget: {
+    notificationCenterEncryption: false,
+  },
+  __v: 0,
+  _id: '633a68831c6bf8ce582c923e',
+  _organizationId: '633a68831c6bf8ce582c9238',
+};
+
+const apiKeys: { key: string }[] = [
```

Comment:
```suggestion
const apiKeys: IApiKey[] = [
```
This should be the right interface for the API keys. Also you would need to include an `userId`. You can double check it here: https://github.com/novuhq/novu/blob/bd7c2490725f031cb832ab8d801ad61ae9e293fc/libs/dal/src/repositories/environment/environment.entity.ts#L1

Reply:
I've updated apiKeys type at 713616c5cb18ba43df64bd7cafffff364fa2b6c9.

I've got a question about importing the interface.

I tried to import `IApiKey` by `` import { IApiKey } from '@novu/dal'``, but the import usage seems to be restricted by eslint. Is this what you expected?

Reply:
Just skip this request for this PR as we have realised it needs some more sensible changes. 👍🏻